clients/admin-ui/src/features/privacy-requests/RequestTableColumns.tsx

```diff
@@ -42,6 +47,21 @@ export const getRequestTableColumns = (revealPII = false) => [
     ),
     header: (props) => <DefaultHeaderCell value="Days left" {...props} />,
   }),
+  ...(hasPlus
+    ? [
+        columnHelper.accessor((row) => row.source, {
+          id: COLUMN_IDS.SOURCE,
+          cell: (props) =>
+            props.getValue() ? (
+              <BadgeCell value={props.getValue()!} />
+            ) : (
+              <DefaultCell value={undefined} />
+            ),
+          header: (props) => <DefaultHeaderCell value="Source" {...props} />,
+          enableSorting: false,
+        }),
+      ]
+    : []),
```

Comment:
Not sure of the best practice here, passing in `hasPlus` vs defining `const { plus: hasPlus } = useFeatures();` locally